entity-framework/ef6/fundamentals/databinding/wpf.md

```diff
@@ -5,9 +5,9 @@ ms.date: "03/31/2020"
 ms.assetid: e90d48e6-bea5-47ef-b756-7b89cce4daf0
 ---
 > [!IMPORTANT]
-> **This document is for WPF on the .NET Framework only**
+> **This document is valid for WPF on the .NET Framework only**
 >
-> This document covers databinding for WPF on the .NET Framework. For WPF on .NET Core, please refer to the [EF Core documentation](~ef/core).
+> This document describes databinding for WPF on the .NET Framework. If your app is built with .NET Core, please refer to the [EF Core documentation](~/ef/core).
```

Comment:
Not sure if this is right. Is using EF Core (as opposed to EF6 on .NET Core) going to help here?

Reply:
Is there a better way to word this? Our guidance for _new_ .NET Core projects is to use EF Core, right? We can look separately at having a databinding section there,  but I'm assuming either new projects (EF6 + .NET Framework) will make sense, OR they may be porting existing EF6 code from .NET Framework to .NET Core in which case this document is irrelevant (because it is a file->new experience).

Reply:
@ajcvickers I revised the language to reference #778